shell-completion/Makefile

```diff
@@ -1,29 +1,31 @@
+MAKEFLAGS += --jobs=$(shell nproc)
 
 .PHONY: command-options clean
 
 all: hledger-completion.bash
 
-hledger-completion.bash: hledger-completion.bash.m4 commands-list.txt query-filters.txt generic-options.txt command-options
+hledger-completion.bash: hledger-completion.bash.m4 query-filters.txt generic-options.txt command-options
 	m4 hledger-completion.bash.m4 > $@
 
 generic-options.txt:
 	hledger -h | ./output-options.sh | sort -u > $@
 
-commands.txt:
```

Comment:
This target is included by m4, so we need it. Should be in the prerequisites for `hledger-completion.bash` too.
Maybe something like:
```make
commands.txt:
	printf "%s\n" $(COMMANDS) > $@
```

Reply:
Yup. I had planned to redo the `m4` handoff so this wasn't necessary, but at as of this PR it still is. Replaced with your suggestion.